app/src/main/java/org/fossasia/phimpme/base/BaseActivity.java

```diff
@@ -18,11 +20,25 @@
 
     protected BottomNavigationView navigationView;
 
+    private int[][] states = new int[][] {
+            new int[] {android.R.attr.state_checked}, // checked
+            new int[] {-android.R.attr.state_checked}, // unchecked
+    };
+
+    private int[] colors = new int[] {
+            Color.WHITE, // checked
+            0 // unchecked set default in onCreate
+    };
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(getContentViewId());
+        colors[1] = getResources().getColor(R.color.bottom_navigation_tabs , getTheme());
```

Comment:
try using this once
    colors[1]  = ContextCompat.getColor(this, R.color.bottom_navigation_tabs);